test/budget.html

```diff
@@ -0,0 +1,56 @@
+---
+title: Sample advanced search for Budget (custom)
+description: Demo page for the Canada.ca Search UI Advanced for Budget
+lang: en
+nositesearch: true
+pageclass: page-type-search
+pageType: recherche
+share: false
+deptfeature: false
+dateModified: 2025-02-24
+css: "../src/connector.css"
+script:
+- src: "../src/connector.js"
+  type: module
+---
+
+<!-- Sorting example -->
+<div class="alert alert-info">
+	<p><strong>Try this out!</strong> Click on the following link to search for "Taxes" with the search results ordered by dates descending, instead of relevance.</p>
+	<p><a href="budget.html?allq=Taxes&sort=date">Sort results by date</a></p>
+	<p>Note: Current implementation of the sorting feature requires server-side code or additional custom JS to make dynamic with the search query.</p>
+</div>
+
+<!--FORM-->
+<form id="gc-searchbox">  
+	<fieldset> 
+		<legend class="h3 mrgn-tp-sm">Find pages with...</legend> 
+		<div class="form-group"> 
+			<label for="advseacon1">all these words:</label> 
+			<input name="allq" class="form-control" id="advseacon1" maxlength="100" data-fusion-query="safe" aria-describedby="gc-pi">
```

Comment:
Info:

If possible, let handle aria via JS to reduce complexity

Reply:
If an enterprise approach doesn't arise from GCWeb in the near future, we'll have to consider planning for this. Thanks for bringing this up!